admin_guide/managing_projects.adoc

```diff
@@ -94,3 +101,55 @@ or:
 To request a new project, fill out the project request form located at
 https://internal.example.com/openshift-project-request.
 ----
+
+== Node Selectors
+
+Node selectors are used in conjunction with labels to control pod placement. 
+
+[NOTE]
+Labels can be assigned link:../install_config/install/advanced_install.html#configuring-node-host-labels[during an advanced installation], or link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[added to a node after installation].
```

Comment:
It's not broken, but, wrap this in `====` if you want to match the guideline style.